src/invoices/InvoiceDetails/Information/Information.js

```diff
@@ -132,6 +133,13 @@ const Information = ({
         <Col xs={6}>
           <BillTo billToId={billTo} />
         </Col>
+
+        <Col xs={3}>
+          <KeyValue
```

Comment:
I think it should be a separate component, like this https://github.com/folio-org/stripes-acq-components/blob/master/lib/Organization/OrganizationValue/OrganizationValue.js

Reply:
good idea, thanks